spec/registry.md

```diff
@@ -907,7 +907,7 @@ are encouraged to track development of these options during Tech Preview:
 
 When an _operand_ or an _option_ value uses a _variable_ annotated,
 directly or indirectly, by a `:datetime` _annotation_,
-its resolved value contains only the implementation-defined datetime value
+its resolved value contains the implementation-defined datetime value
```

Comment:
as with the `:number` PR, I think "the implementation-defined" should consistently be "an implementation-defined" and for the same reason.

This one says "datetime", the others say "date/time"

Reply:
```suggestion
its resolved value contains an implementation-defined date/time value
```